lib/cassandra/cassandra-facade.js

```diff
@@ -0,0 +1,145 @@
+'use strict';
+
+const cassandra = require('cassandra-driver');
+
+let client;
+
+class CassandraFacade {
```

Comment:
Substitute facade for a query helper that takes a Cassandra client as a parameter for the methods (let the users create their own Cassandra client)